src/treeView.ts

```diff
@@ -80,6 +80,10 @@ export class TreeDataProvider implements vscode.TreeDataProvider<TreeItem> {
         path: "/" + spec.name + "/" + spec.state,
       });
     } else {
+      if (taskItem.label?.toString().startsWith("Run Guideline Enforcer")) {
+        taskItem.iconPath = new vscode.ThemeIcon("symbol-ruler");
```

Comment:
I would rather have a `circle-filled` icon like other activated tasks

Reply:
I prefer to keep a different icon, and let the circle to sub-commands.
If needed later, wa can transform this into a group (with its icon), and sub-items will have the circle one